preinstall.js

```diff
@@ -35,7 +36,7 @@ if (process.argv.indexOf('--print-lib') > -1) {
       console.log('../libsodium/Build/ReleaseDLL/' + warch + '/libsodium.lib')
       break
     default:
-      process.exit(1)
+      throw new Error('Unsupported platform: ' + os.platform())
```

Comment:
We have two other options here:

## `console.log()`

```javascript
console.log('Unsupported platform: ' + os.platform())
process.exit(1)
```

## No output

I was confused when the process exited without any output, but maybe there's a good reason to do it that way?